source/features/center-reactions-popup.css

```diff
@@ -1,8 +1,9 @@
 .dropdown-menu-reactions.dropdown-menu-reactions { /* Double selector to override .d-flex on Chrome */
-	top: 100% !important;
-	left: 50% !important;
-	right: auto !important;
-	transform: translate(-50%) !important;
+	top: 100%;
+	left: 50%;
+	right: auto;
+	bottom: auto; /* #5957 */
+	translate: -50%;
```

Comment:
💙

Reply:
@fregante wdyt about (@)FloEdelmann's remark?

> I think the bottom emoji selector dropdown should open to the top (like GitHub's native one). When you scroll just so that the emoji selector is the last visible thing on the page, you'd otherwise have to scroll again to click the dropdown.